test/integration/targets/k8s/tasks/main.yml

```diff
@@ -2,13 +2,19 @@
   pip:
     name: openshift
 
+# TODO: This is the only way I could get the kubeconfig, I don't know why. Running the lookup outside of debug seems to return an empty string
```

Comment:
This is really stupid, but I can't figure out how to get that environment variable to the lookup plugin other than this. Would love some guidance.